api/lib/infrastructure/jobs/cpf-export/schedule-cpf-jobs.js

```diff
@@ -25,7 +25,11 @@ const scheduleCpfJobs = async function (pgBoss) {
 export { scheduleCpfJobs };
 
 async function _processJob(job, handler, params) {
-  await handler({ ...params, job, logger: buildLogger(job) });
+  try {
+    await handler({ ...params, job, logger: buildLogger(job) });
+  } catch (err) {
+    logger.error(err);
+  }
```

Comment:
⚠️ Ici on devient la limite de l'erreur. À voir selon "qui" appelle cette fonction.

Reply:
L'appelant est juste au-dessus, `scheduleCpfJobs`, mais il schedule uniquement et n'est pas la fonction executant du jobs sous-jacent.
Actuellement les **unhandled** erreurs étaient juste pas loggées du coup (la stacktrace), à cause de la surcharge de `buildLogger`.

Contrairement aux appels HTTP de l'API, sur les jobs il y a pas de default global handler. Cette fonction me semble être le meilleur candidat actuel du fait.

Verrais-tu la capacité de mettre cela plus haut (sans développer tout un handler global d'erreur qui me semble inexistant actuellement) ?

Reply:
S'il n'y a pas de handler global à l'exécution des jobs pgBoss (et je sais pas si ça a un sens), ici ça me parait ok.
Mais dans l'hypothèse où tu as une aggregation plus haut, le plus safe c'est de rethrow:

```suggestion
  } catch (err) {
    logger.error(err);
    throw err;
  }
```

Reply:
Done